cartridges/app_custom_slots/cartridge/scripts/checkout/checkoutHelpers.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+var base = module.superModule;
+
+var Order = require('dw/order/Order');
+var Resource = require('dw/web/Resource');
+var Site = require('dw/system/Site');
+
+/**
+ * Sends a confirmation to the current user
+ * @param {dw.order.Order} order - The current user's order
+ * @param {string} locale - the current request's locale id
+ * @returns {void}
+ */
+
+function sendConfirmationEmail(order, locale, products) {
```

Comment:
Should be reworked so that products are taken within function body and not getting as a param.